samples/popup/popup.cpp

```diff
@@ -109,8 +109,13 @@ SimpleTransientPopup::SimpleTransientPopup( wxWindow *parent, bool scrolled )
                                               wxBORDER_NONE |
                                               wxPU_CONTAINS_CONTROLS )
 {
+    wxColour colour(*wxLIGHT_GREY);
+
+    if ( wxSystemSettings::GetAppearance().IsDark() )
+        colour.Set(90, 90, 90); // dark grey
+
```

Comment:
It would probably be better to use `SelectLightDark()` here, as it's shorter and also better practice, I think.

Reply:
Thanks, I somehow missed the function.